alfresco-transform-core-aio/alfresco-transform-core-aio-boot/pom.xml

```diff
@@ -187,6 +197,23 @@
                                 </image>
                             </images>
                         </configuration>
+                        <executions>
+                            <execution>
+                                <id>before-integration-tests</id>
+                                <phase>pre-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                    <goal>start</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>after-integration-tests</id>
+                                <phase>post-integration-test</phase>
+                                <goals>
+                                    <goal>stop</goal>
+                                </goals>
+                            </execution>
+                        </executions>
```

Comment:
This will need to be added to each of the other T-engines' pom.xml files too. In a similar place as this. Currently the images are not being started in the integration tests. 
It might be worth adding this entire profile (docker-it-setup) to the super pom.xml as they appear to all be identical, with exception of the t-engine image name. This can then be overridden in each individual projects' pom.xml using the `image.name` property. (which I think they all already have)